corpus/templates/pages/sig.html

```diff
@@ -0,0 +1,57 @@
+{% extends 'base.html' %}
+{% load static %}
+
+{% block title %}
+Home
+{% endblock %}
+
+{% block style %}
+<link rel="stylesheet" href="{% static 'css/index.css' %}" />
```

Comment:
Is there anything specific from `index.css` that this page is using?

Reply:
nope, reverted